.env.local.template

```diff
@@ -0,0 +1,3 @@
+NEXT_PUBLIC_SUPABASE_URL=
```

Comment:
This file might become obsolete with the addition of `global.d.ts` which can define type of env

Reply:
The point of this file is to tell new developers that you do need a `.env.local` file and get them started.